frescobaldi_app/job/lilypond.py

```diff
@@ -102,9 +102,10 @@ def __init__(self, doc, args=None, title=""):
                 input=input,
                 decode_errors='replace',
                 directory=directory,
-                environment={
-                    'LD_LIBRARY_PATH': self.lilypond_info.libdir()
-                },
+                environment=(
+                    {'LD_LIBRARY_PATH': libdir}
+                    if (libdir := self.lilypond_info.libdir())
```

Comment:
question: Just double-checking - the walrus operator is available since 3.8 and we require 3.8?